browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -272,11 +274,12 @@ RewardsDOMHandler::~RewardsDOMHandler() {
 
 void RewardsDOMHandler::RegisterMessages() {
 #if defined(OS_ANDROID)
+  // TODO(sergz) figure out why we have link error here
```

Comment:
?

Reply:
Currently we have link error on that place with brave-core based build, extra dependency or something. I just left it commented for now as it will be figured out in the context of web-ui issue https://github.com/brave/brave-browser/issues/6526

Reply:
ok